### ISSUE TITLE
What HTTP code should api-driver return when DID is not found?

### ISSUE THREAD

**Issue by bozzzzo** [decentralized-identity/universal-resolver#392]
According to [api-driver.yml](../blob/main/swagger/api-driver.yml) the driver can respond to `/identifiers/{identifier}` only with HTTP codes 200, 400, 410 or 500. The example resolver returns 404 for not found DID.

Add missing valid HTTP response codes to the `api-driver.yml`

**Comment by peacekeeper** [comment]
I agree, the .yml file should be updated accordingly. The DID Resolution specification also mentions 404: https://w3c-ccg.github.io/did-resolution/#bindings-https
